snuba/datasets/discover.py

```diff
@@ -282,6 +283,7 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
         discover_source = self.get_dataset_schemas().get_read_schema().get_data_source()
 
         return [
+            BasicFunctionsProcessor(),
```

Comment:
Do you see apdex, impact and the other discover related functions being added as a separate processor?

Reply:
yes. apdex is going to be a query processor